action-allowedlist/src/entrypoint.ps1

```diff
@@ -3,18 +3,31 @@
 # The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 # See the LICENSE and NOTICES files in the project root for more information.
 
-Import-Module /src/action-allowedlist.psm1 -DisableNameChecking
+Import-Module /src/action-loader.psm1 -DisableNameChecking
+Import-Module /src/allowed-action-analyzer.psm1 -DisableNameChecking
+Import-Module /src/deprecated-action-analyzer.psm1 -DisableNameChecking
 
 $actionsFound = LoadAllUsedActions -RepoPath $pwd
 $unapproved = CheckIfActionsApproved -outputs $actionsFound
-$jsonObject = ($unapproved | ConvertTo-Json)
-$jsonObject | out-file ./actions.json
-Write-Output "::set-output name=actions::'$jsonObject'"
+$jsonUnapprovedResults = ($unapproved | ConvertTo-Json)
+$jsonUnapprovedResults | out-file ./unapproved-actions.json
+
+Write-Output "name=unapproved-actions::<<EOF" >> $GITHUB_OUTPUT
+Write-Output "$jsonUnapprovedResults" >> $GITHUB_OUTPUT
+Write-Output "EOF" >> $GITHUB_OUTPUT
+
+$deprecated = CheckIfActionsDeprecated -outputs $actionsFound
+$jsonDeprecatedResults = ($deprecated | ConvertTo-Json)
+$jsonDeprecatedResults | out-file ./deprecated-actions.json
+
+Write-Output "name=deprecated-actions::<<EOF" >> $GITHUB_OUTPUT
+Write-Output "$jsonDeprecatedResults" >> $GITHUB_OUTPUT
+Write-Output "EOF" >> $GITHUB_OUTPUT
+
 
 if ($unapproved.Count -gt 0) {
-    Write-Error "Repo contains unapproved actions!"
+    Write-Error "Repo contains one or more action that is not approved for use."
```

Comment:
I changed this text while trying to debug and understand if [my test](https://github.com/stephenfuqua/Meadowlark/actions/runs/3833122168/jobs/6524225683) is using old or new code. I had to do mess with my [on-pullrequest](https://github.com/stephenfuqua/Meadowlark/blob/main/.github/workflows/on-pullrequest.yml) in order to test this, manually inserting the workflow steps instead of calling the reusable workflow - because the reusable workflow was hard-coded to using the real Ed-Fi-Actions repository.

So now I have a valid test. The run linked above fails because I have the deprecated dorny-reporter in it. I do not get a deprecation warning - I just get a failure. It seems like the old dorny test reporter needs to be in the allowed list as well.